radicle-cli/src/commands/untag.rs

```diff
@@ -49,7 +50,7 @@ impl Args for Options {
                 Value(ref val) if id.is_none() => {
                     let val = val.to_string_lossy();
                     let Ok(val) = cob::ObjectId::from_str(&val) else {
-                        return Err(anyhow!("invalid issue or patch ID '{}'", val));
+                        return Err(anyhow!("invalid id for issue or patch '{}'", val));
```

Comment:
Use the issue-id parsing function here

Reply:
you mean cob-id one?

Reply:
used `term::args::cob` https://github.com/radicle-dev/heartwood/pull/445/commits/6d0b338c6920de263d264d7bac5e5c819bc80511#diff-d7c8d73a94b519d345b3f732f2b8faebaf10533ec11cfb7035688c3c3212b442R51